stories/atoms/components/button/Button.stories.mdx

```diff
@@ -31,196 +28,343 @@ import { Icon } from 'ui/atoms/icon/Icon'
+and cursor.
 
 <Canvas>
   <Story name="Disabled">
-    {args => (
-      <div
-        style={{
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-          gap: '25px',
-          flexWrap: 'wrap'
-        }}
-      >
-        <Button disabled variant="primary" label="I'm disabled" size="medium" />
-      </div>
-    )}
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'center'
+      }}
+    >
+      <Button disabled variant="primary" label="I'm disabled" size="medium" />
+    </div>
   </Story>
 </Canvas>
 
-## Content
+## Left / Right Icon
 
-The content can be customized to fit users needs through its properties `label`, `icon`,
-`leftIcon` and `rightIcon`.
+`leftIcon` and `rightIcon` are auxilliary elements giving great freedom to embellish the button
+together with the `label`.
 
-- `label` is the only obligatory property and is used to clearly indicate what action will be triggered, through the
-  tooltip and text.
-- `icon` sets the icon for the **variant** 'icon'.
-- `leftIcon` and `rightIcon` are auxilliary elements giving great freedom to embellish the button
-  together with the label.
+<Canvas>
+  <Story name="Left / right Icon">
+    {() => {
+      const { theme } = useTheme()
+      const invert = theme === 'light'
+      return (
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            flexWrap: 'wrap',
+            gap: '50px'
+          }}
+        >
+          <Button
+            leftIcon={<Icon name="wallet" size={22} invertColor={invert} />}
+            label="Get wallet address"
+          />
+          <Button
+            label="Add wallet"
+            rightIcon={<Icon name="add" size={22} invertColor={invert} />}
+          />
+        </div>
+      )
+    }}
+  </Story>
+</Canvas>
+
+## OnClick
+
+`onClick` is what makes the magic happen, without it the button has no corresponding action.
+It is triggered when a click is registered on the button.
 
 <Canvas>
-  <Story name="Content">
-    {args => (
-      <div
-        style={{
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-          gap: '25px',
-          flexWrap: 'wrap'
-        }}
-      >
-        <Button icon={<Icon name="wallet" size={22} />} label="wallet" variant="icon" />
-        <Button icon={<Icon name="wallet" size={22} />} label="wallet" variant="icon" />
-        <Button leftIcon={<Icon name="wallet" size={22} />} label="Get wallet address" />
-        <Button
-          label="Add wallet"
-          leftIcon={<Icon name="wallet" size={22} />}
-          rightIcon={<Icon name="add" size={22} />}
-        />
-      </div>
-    )}
+  <Story name="OnClick">
+    {() => {
+      const [opened, setOpened] = useState(false)
+      const handleOpened = isOpened => () => {
```

Comment:
I'm not a big fan of this syntax when parameter is not mandatory for the method.
I think we can be more concise:
```suggestion
      const handleOpen = () => {
        setIsOpened(prevState => !prevState)
```